src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -85,7 +85,9 @@ def create(input_dataset, target, feature=None, validation_set='auto',
 
     pretrained_model_url : string optional
         A URL to the pretrained model that must be used for a warm start before
-        training.
+        training. By default, we load the pretrained model we trained and uploaded
+        on https://docs-assets.developer.apple.com/turicreate/models/
```

Comment:
I wouldn't mention the URL or that we trained it. I would just say "By default, we load the pretrained model."

Reply:
Fixed.